src/Avalonia.Controls.ColorPicker/Themes/Fluent/ColorView.xaml

```diff
@@ -0,0 +1,643 @@
+                                          MinHue="{TemplateBinding MinHue}"
+                                          MaxHue="{TemplateBinding MaxHue}"
+                                          MinSaturation="{TemplateBinding MinSaturation}"
+                                          MaxSaturation="{TemplateBinding MaxSaturation}"
+                                          MinValue="{TemplateBinding MinValue}"
+                                          MaxValue="{TemplateBinding MaxValue}"
+                                          Shape="{TemplateBinding ColorSpectrumShape}"
+                                          HorizontalAlignment="Stretch"
+                                          VerticalAlignment="Stretch" />
+                <primitives:ColorSlider x:Name="ColorSpectrumAlphaSlider"
+                                        AutomationProperties.Name="Alpha Component"
+                                        Grid.Column="2"
+                                        Orientation="Vertical"
+                                        ColorModel="Hsva"
+                                        ColorComponent="Alpha"
+                                        HsvColor="{Binding HsvColor, ElementName=ColorSpectrum}"
+                                        HorizontalAlignment="Center"
+                                        VerticalAlignment="Stretch"
+                                        Margin="12,0,0,0"
+                                        IsEnabled="{TemplateBinding IsAlphaEnabled}">
+                  <primitives:ColorSlider.IsVisible>
+                    <MultiBinding Converter="{x:Static BoolConverters.And}">
+                      <Binding RelativeSource="{RelativeSource TemplatedParent}"
+                               Path="IsAlphaVisible" />
+                    </MultiBinding>
+                  </primitives:ColorSlider.IsVisible>
+                </primitives:ColorSlider>
+              </Grid>
+            </TabItem>
+            <!-- Palette Tab -->
+            <TabItem IsVisible="{TemplateBinding IsColorPaletteVisible}">
+              <TabItem.Header>
+                <Border Height="{DynamicResource ColorViewTabStripHeight}">
+                  <PathIcon Width="20"
+                            Height="20"
+                            Data="{DynamicResource ColorViewPaletteIconGeometry}" />
+                </Border>
+              </TabItem.Header>
+              <ListBox Items="{TemplateBinding PaletteColors}"
+                       SelectedItem="{Binding Color, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
+                       UseLayoutRounding="False"
+                       Margin="12">
+                <ListBox.Styles>
+                  <Style Selector="ListBoxItem">
+                    <Setter Property="Background" Value="Transparent" />
+                    <Setter Property="Padding" Value="0" />
+                    <Setter Property="Template">
```

Comment:
The template here probably should be pulled out as a property on ColorView itself. That would allow changing to elliptical color swatches or something without re-templating.

Reply:
I think for now it's flexible enough as it is. User can override ListBoxItem.Template with nested selectors.

Reply:
Ok, well I won't add the property then for now and will just keep it in mind.